src/lazy/encoder/text/v1_0/value_writer.rs

```diff
@@ -216,7 +215,7 @@ impl<'value, W: Write> Sealed for TextValueWriter_1_0<'value, W> {}
 
 /// Helper type that is home to information and behavior common to the list writer, s-expression writer,
 /// and struct writer.
-struct TextContainerWriter_1_0<'a, W: Write> {
+pub(crate) struct TextContainerWriter_1_0<'a, W: Write> {
```

Comment:
🗺️ Because the 1.1 text writer mostly delegates to the 1.0 text writing types, many fields and types are now `pub(crate)`.